js/cartodb_lib.js

```diff
@@ -0,0 +1,361 @@
+    var results = $('#results-list');
+
+    if ((CartoDbLib.whereClause == ' WHERE the_geom is not null AND ') || (CartoDbLib.whereClause == ' WHERE the_geom is not null ')) {
+      CartoDbLib.whereClause = '';
+    }
+
+    results.empty();
+    sql.execute("SELECT " + CartoDbLib.fields + " FROM " + CartoDbLib.tableName + CartoDbLib.whereClause + " ORDER BY name")
+      .done(function(listData) {
+        var obj_array = listData.rows;
+
+        if (listData.rows.length == 0) {
+          results.append("<p class='no-results'>No results. Please broaden your search.</p>");
+        }
+        else {
+          var row_content;
+          $.get( "/templates/table-row.ejs", function( template ) {
+              for (idx in obj_array) {
+
+                row_content = ejs.render(template, {obj: obj_array[idx]});
+
+                results.append(row_content);
+              }
+            });
+          }
+    }).error(function(errors) {
+      console.log("errors:" + errors);
+    });
+  },
+
+  getResults: function() {
+    var sql = new cartodb.SQL({ user: CartoDbLib.userName });
+
+    sql.execute("SELECT count(*) FROM " + CartoDbLib.tableName + CartoDbLib.whereClause)
+      .done(function(data) {
+        CartoDbLib.resultsCount = data.rows[0]["count"];
+        CartoDbLib.results_div.update(CartoDbLib.resultsCount);
+        $('#list-result-count').html(CartoDbLib.resultsCount + ' locations found')
+      }
+    );
+  },
+
+  modalPop: function(data) {
+
+    var modal_content;
+    $.get( "/templates/popup.ejs", function( template ) {
+        modal_content = ejs.render(template, {obj: data});
+        $('#modal-pop').modal();
+        $('#modal-main').html(modal_content);
+        $.address.parameter('modal_id', data.id);
+      });
+  },
+
+  clearSearch: function(){
+    if (CartoDbLib.sublayer) {
+      CartoDbLib.sublayer.remove();
+    }
+    if (CartoDbLib.centerMark)
+      CartoDbLib.map.removeLayer( CartoDbLib.centerMark );
+    if (CartoDbLib.radiusCircle)
+      CartoDbLib.map.removeLayer( CartoDbLib.radiusCircle );
+  },
+
+  findMe: function() {
+    // Try W3C Geolocation (Preferred)
+    var foundLocation;
+
+    if(navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function(position) {
+        foundLocation = new google.maps.LatLng(position.coords.latitude,position.coords.longitude);
+        CartoDbLib.addrFromLatLng(foundLocation);
+      }, null);
+    }
+    else {
+      alert("Sorry, we could not find your location.");
+    }
+  },
+
+  addrFromLatLng: function(latLngPoint) {
+    geocoder.geocode({'latLng': latLngPoint}, function(results, status) {
+      if (status == google.maps.GeocoderStatus.OK) {
+        if (results[1]) {
+          $('#search-address').val(results[1].formatted_address);
+          CartoDbLib.doSearch();
+        }
+      } else {
+        alert("Geocoder failed due to: " + status);
+      }
+    });
+  },
```

Comment:
Similarly, it seems like this method is only used in `findMe()`.

Reply:
good catch